_docs/tasks/basic-access-control.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Enabling Simple Access Control
+overview: This task shows how to use Istio to control access to a service.
+          
+order: 41
+
+layout: docs
+type: markdown
+---
+
+This task shows how to use Istio to control access to a service.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/tasks/installing-istio.html).
+
+* Deploy the [bookinfo](/docs/samples/bookinfo.html) sample application.
+
+* Initialize the application version routing by either first doing the
+  [request routing](/docs/tasks/request-routing.html) task or by running following
+  commands:
+  
+  ```bash
+  istioctl create -f route-rule-all-v1.yaml
+  istioctl replace -f route-rule-reviews-v2-v3.yaml
+  ```
+* Ensure that you can use [istioctl mixer](/docs/reference/commands/istioctl/istioctl_mixer.html#synopsis) by setting up port forwarding if needed.
+
+## Access control using `denials` 
+
+Using Istio you can control access to a service based on any attributes that are available within Mixer.
+This simple form of access control is based on conditionally denying requests using Mixer selectors.
+
+Consider the [bookinfo](/docs/samples/bookinfo.html) sample application where `ratings` service is accessed by multiple versions
```

Comment:
bookinfo -> BookInfo
URL -> ({{home}}/docs/samples/bookinfo.html)
`ratings` -> the `ratings`